src/server.ts

```diff
@@ -19,8 +19,16 @@
  */
 import axios from 'axios';
 import semver, { SemVer } from 'semver';
+import { LogLevel, log } from './logging';
 
 export async function fetchServerVersion(serverUrl: string): Promise<SemVer> {
-  const { data } = await axios.get(`${serverUrl}/api/server/version`);
-  return semver.coerce(data) ?? data; // TODO: Do we want to fail when we can't get the server version? That'd make sense
+  try {
+    log(LogLevel.DEBUG, 'Fetch URL: ', `${serverUrl}/api/server/version`);
+    const { data } = await axios.get(`${serverUrl}/api/server/version`);
+    log(LogLevel.DEBUG, 'Server version:', data);
+    return semver.coerce(data) ?? data; // TODO: Do we want to fail when we can't get the server version? That'd make sense
```

Comment:
Should we handle the case where the server returns data but the version can't be parsed? That can happen when SQ is behind a reverse proxy and it returns garbage data like HTML, I'm wondering if we should fail explicitly in that case as well?

Reply:
we can consider it, but I added this because I ran into this when SQ wasn't running and would fail. Since this is the first request made, we should consider also some edge cases for users who misconfigure their scanner (trailing slash I think was my case -> HTTP://localhost:9000/ mapped to HTTP://localhost:9000//api/server/version which is invalid)